snprc_ehr/resources/schemas/snprc_ehr.xml

```diff
@@ -1582,6 +1582,10 @@
             <column columnName="BirthDate">
                 <datatype>datetime</datatype>
             </column>
+            <column columnName="BirthCode">
+                <datatype>integer</datatype>
+                <defaultValue>1</defaultValue>
```

Comment:
Are we going to add a tsv lookup table for the BirthCode? If so, the fk needs to be added to the BirthCode metadata.